### PR TITLE
August changelog

### LINKED ISSUE #2730: August/September Changelog Candidates
Add August platform changes and new features to be included in the August/September Changelog.

### PR DESCRIPTION
**Pull request by alexfornuto**
Closes #2730 

## Effect
PR includes the following changes:
- August Changelog

### CODE REVIEW

File: source/_changelogs/2017-08-01-August.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: August 2017
+changelog: true
+---
+
+## Platform Improvements
+### Global CDN Surrogate Key Cache Clearing API Improvement
```
Comment (ari-gold) [inline source/_changelogs/2017-08-01-August.md]: How about including links to https://wordpress.org/plugins/pantheon-advanced-page-cache/ and https://drupal.org/project/pantheon_advanced_page_cache ?

File: source/_changelogs/2017-08-01-August.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+title: August 2017
+changelog: true
+---
+
+## Platform Improvements
+### Global CDN Surrogate Key Cache Clearing API Improvement 
+We improved cache clearing behavior for a large number of surrogate keys or cache tags. The fix was made internally by Pantheon and no action is required if you are already running WordPress or Drupal versions of Pantheon Advanced Page Cache.
+
+**See Also:** Pantheon Advanced Page Cache for [WordPress](https://github.com/pantheon-systems/documentation/pull/2863/files#r137361525) and [Drupal](https://www.drupal.org/project/pantheon_advanced_page_cache)
```
Comment (alexfornuto) [inline source/_changelogs/2017-08-01-August.md]: @ari-gold Like this?

**Comment by rachelwhitton** [comment]
+1

File: source/_changelogs/2017-08-01-August.md (hunk 2 above)
Comment (ari-gold) [reply source/_changelogs/2017-08-01-August.md]: typo in the WP link?

**Pull request merged**